Sources/xcodeproj/Objects/BuildPhase/PBXBuildFile.swift

```diff
@@ -17,6 +17,19 @@ public final class PBXBuildFile: PBXObject {
         }
     }
 
+    /// Product reference.
+    var productReference: PBXObjectReference?
+
+    /// Product.
+    public var product: XCSwiftPackageProductDependency? {
```

Comment:
Would this just be a plain `PBXObject` or `PBXContainerItem`?

Reply:
Looks like the `XCSwiftPackageProductDependency` is a reference for a `PBXTargetDependency`. As far as I am aware they introduced a new type `XCRemoteSwiftPackageReference` which contains things like the package url and versioning information. Checkout the issue where there's more information dumped: https://github.com/tuist/xcodeproj/issues/438

Reply:
> Would this just be a plain PBXObject or PBXContainerItem?

Not sure @yonaskolb . Since the attribute has been introduced to refer to package product dependencies, I'd keep the type and make it more generic in the future if we see that Apple is using it for something else.